app/src/main/java/tech/relaycorp/gateway/domain/endpoint/RegisterEndpoint.kt

```diff
@@ -1,16 +1,36 @@
 package tech.relaycorp.gateway.domain.endpoint
```

Comment:
I had to issue this cert as part of #72, so I think we can undo the changes to this file (the new method doesn't seem to be used anyway).